src/main/java/frc/robot/Constants.java

```diff
@@ -245,4 +246,21 @@ public static final class AutoConstants {
                 kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
     }
 
+    public static final class kOI {
+        public static final int DRIVE_TRANSLATION_Y = XboxController.Axis.kLeftY.value;
+        public static final int DRIVE_TRANSLATION_X = XboxController.Axis.kLeftX.value;
+        public static final int DRIVE_ROTATE = XboxController.Axis.kRightX.value;
+
+        public static final int RUN_INTAKE = XboxController.Button.kLeftBumper.value;
+        public static final int EJECT_INTAKE = XboxController.Button.kX.value;
+
+        public static final int BACK_INDEXER = XboxController.Button.kY.value;
+
+        public static final int ZERO_GYRO = XboxController.Button.kA.value;
+        public static final int ZERO_SHOOTER_HOOD = XboxController.Button.kA.value;
+
+        public static final int AUTO_SHOOT = XboxController.Button.kRightBumper.value;
+        public static final int FENDER_SHOOT = XboxController.Button.kB.value;
+    }
```

Comment:
are we doing scream case (`SCREAM_CASE`) or not? if we are, why only add these ones?